src/components/content/order/orderStatus/OrderSubmitResultDetails.tsx

```diff
@@ -3,10 +3,43 @@
  * SPDX-FileCopyrightText: Huawei Inc.
  */
 
+import { Typography } from 'antd';
+import { CheckOutlined, CopyOutlined } from '@ant-design/icons';
+import React from 'react';
+import { myServicesRoute } from '../../../utils/constants';
+import { createSearchParams, useNavigate } from 'react-router-dom';
+
 function OrderSubmitResultDetails({ msg, uuid }: { msg: string | JSX.Element; uuid: string }): JSX.Element {
+    const { Paragraph } = Typography;
+    const navigate = useNavigate();
     return (
         <>
-            Request ID: <b>{uuid}</b>
+            <div className={'service-instance-detail-position'}>
+                <span className={'service-instance-detail-position-size'}>Service ID:</span>
+                <Paragraph
+                    copyable={{
+                        text: String(uuid),
+                        icon: [
+                            <CopyOutlined className={'show-details-typography-copy'} />,
+                            <CheckOutlined className={'show-details-typography-copy'} />,
+                        ],
+                    }}
+                >
+                    <span
+                        onClick={() => {
+                            navigate({
+                                pathname: myServicesRoute,
+                                search: createSearchParams({
+                                    serviceId: uuid,
+                                }).toString(),
+                            });
+                        }}
+                        className={'show-details-typography-copy-info'}
+                    >
+                        {uuid}
+                    </span>
+                </Paragraph>
+            </div>
             <br />
```

Comment:
in the previous version -> the line break was after the text 'Processing Status'